server.jsx

```diff
@@ -3,22 +3,47 @@
 import { serve } from "https://deno.land/std@0.121.0/http/server.ts";
 import { h, Helmet, renderSSR } from "nano-jsx";
 import { Package } from "./lib/package.js";
-import { FeaturedPackages } from "./lib/featured-packages.js";
+import { Home } from "./lib/home.js";
 import { pageServingHeaders, renderMarkdownContent } from "./utils.js";
-import { featuredPackages } from "./lib/featured-packages-list.js";
+import { FEATURED_PACKAGES } from "./lib/featured-packages-list.js";
 
 const staticResources = {
   "/style.css": { path: "./style.css", contentType: "text/css; charset=utf-8" },
+  "/package/style.css": {
```

Comment:
Was checking to solve this, can we regex the URL and pick the last filename and pass to this map ?

Reply:
thanks for reviewing. must be fixed now